### PR TITLE
Bug 2061995: Use device location provided by the user

### PR DESCRIPTION
**Pull request by gnufied**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=2061995

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,9 +1,10 @@
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.9 AS builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.11 AS builder
```
Comment (jsafrane) [inline Dockerfile:1]: It should use 4.10 images.
Perhaps we should delete unused Dockerfiles (without .rhel7 suffix) eventually.

Comment (gnufied) [reply Dockerfile:1]: oops sorry I just blindly cherry-picked. yeah we could delete these. currently `sync_bundle` script refers those images and that needs to be updated too.

Comment (gnufied) [reply Dockerfile]: fixed

**Comment by gnufied** [comment]
@chao007 can you add cherry-pick approved?

/assign @chao007

**Pull request merged**